staging/src/k8s.io/client-go/copy.sh

```diff
@@ -18,17 +18,36 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+VERIFYONLY=false
+while getopts ":v" opt; do
+  case $opt in
+    v)
+      VERIFYONLY=true
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
+      ;;
+  esac
+done
+
 # PREREQUISITES: run `godep restore` in the main repo before calling this script.
 RELEASE="1.5"
 CLIENTSET="release_1_5"
-MAIN_REPO_FROM_SRC="${1:-"k8s.io/kubernetes"}"
+MAIN_REPO_FROM_SRC="k8s.io/kubernetes"
 MAIN_REPO="${GOPATH%:*}/src/${MAIN_REPO_FROM_SRC}"
-CLIENT_REPO_FROM_SRC="${2:-"k8s.io/client-go/${RELEASE}"}"
+CLIENT_REPO_FROM_SRC="k8s.io/client-go/${RELEASE}"
+CLIENT_REPO_TEMP_FROM_SRC="k8s.io/client-go/_tmp"
 CLIENT_REPO="${MAIN_REPO}/staging/src/${CLIENT_REPO_FROM_SRC}"
-CLIENT_REPO_TEMP="${CLIENT_REPO}"/_tmp
+CLIENT_REPO_TEMP="${MAIN_REPO}/staging/src/${CLIENT_REPO_TEMP_FROM_SRC}"
 
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
+cleanup() {
+    rm -rf "${CLIENT_REPO_TEMP}"
+}
+
+trap cleanup EXIT SIGINT
```

Comment:
Oh, right: `rm` apparently does not return a non-zero code if the directory does not exist.